prisma-fmt/src/text_document_completion.rs

```diff
@@ -85,19 +86,47 @@ fn push_ast_completions(ctx: CompletionContext<'_>, completion_list: &mut Comple
         .relation_mode()
         .unwrap_or_else(|| ctx.connector().default_relation_mode());
 
-    match ctx.db.ast(ctx.initiating_file_id).find_at_position(position) {
+    let find_at_position = ctx.db.ast(ctx.initiating_file_id).find_at_position(position);
```

Comment:
In the process of updating `find_at_position` for `references` I broke completions for referential actions. However, due to the changes in `find_at_position` we are now able to offer completions on partial values:


https://github.com/prisma/prisma-engines/assets/29753584/3075f0f3-ef0c-4f27-a585-1fca0ab122a6